### DISCUSSION TITLE
Upgrade to 0.10.2 from 0.6.0

### DISCUSSION THREAD

**Discussion by shashikantpawar21** [Eventuous/eventuous#129] [Q&A]
We are upgrading from **0.6.0 to 0.10.2** 

We did the changes of removing the aggregate to comply with new version 
Somehow, we also had to change some event definitions 
Locally the changes are tested properly 

What would risk to move to staging environment which is already on 0.6.0 version 

Will the data mismatch happen because event definition changes ?

Did anyone done this upgrade .. would want to know the suggestion related to same ?

**Comment by alexeyzimarev** [comment]
There's no requirement to change event contracts for the new version. You can remove the aggregate id from events, but only if you want to.
